generator/scenarios/EVMTransfer.go

```diff
@@ -57,6 +57,14 @@ func (s *EVMTransferScenario) CreateTransaction(config *config.LoadConfig, scena
 		Data:      nil,                     // No data for simple transfer
 	}
 
+	if s.config != nil && s.config.Settings != nil && s.config.Settings.GasPicker != nil {
+		var err error
+		tx.Gas, err = s.config.Settings.GasPicker.GenerateGas()
```

Comment:
So tx.gas basically will override the actual gas limit, but how does that impact priority?

Reply:
I believe this translates- for cosmos [here](https://github.com/sei-protocol/sei-cosmos/blob/c45dd4a11e4ba4e612783c06bd032fec2d4e3aee/x/auth/ante/validator_tx_fee.go#L67) is the code. For EVM I also need to parameterise fee and tip caps. Thanks for pointing that out Yiming